### PR TITLE
A basic check for known-conflicting files

### PR DESCRIPTION
**Pull request by invisnet**
So far only `composer.json` - seems to work.

### CODE REVIEW

File: lib/admin-page.php [hunk 1]
```diff
@@ -449,6 +449,36 @@ function classicpress_check_can_migrate() {
...
+		_e(
+			'Conflicting files detected.',
+			'switch-to-classicpress'
+		);
+		echo "</strong>\n<br>\n";
+		_e(
+			'If you have JavaScript enabled, you can see a list of conflicting files <strong>in your browser console</strong>.',
+			'switch-to-classicpress'
+		);
+		echo "\n<script>console.log( 'modified core files:', ";
```
Comment (nylen) [inline lib/admin-page.php]: message here needs to be updated to say "conflicting files"

File: lib/admin-page.php [hunk 2]
```diff
@@ -449,6 +449,36 @@ function classicpress_check_can_migrate() {
...
+	}
+	if ( empty( $conflicting_files ) ) {
+		_e(
+			'No conflicting files were found.',
+			'switch-to-classicpress'
+		);
+	} else {
+		echo '<strong class="cp-emphasis">';
+		_e(
+			'Conflicting files detected.',
```
Comment (nylen) [inline lib/admin-page.php]: a bit more explanation here?  "Conflicting files detected.  These are files that will be <strong>overwritten</strong> by the migration."

Comment (invisnet) [reply lib/admin-page.php]: The logic now is that conflicting files block migration - there's every chance we cannot overwrite them and there's no way to resolve the conflict automatically.

Comment (nylen) [reply lib/admin-page.php]: OK, that makes sense, the problem is this doesn't explain what "conflicting files" means or how to resolve the issue.  Maybe this then?

> Conflicting files detected. These are files that would be **overwritten** during migration.

Comment (invisnet) [reply lib/admin-page.php]: Works for me.

**Comment by nylen** [comment]
2019-02-18t15 30 07-05 00

**Pull request merged**